tracker/gioui/numericupdown.go

```diff
@@ -74,11 +80,22 @@ func NumericUpDown(th *material.Theme, number *NumberInput, tooltip string) Nume
 		Tooltip:         Tooltip(th, tooltip),
 		Width:           unit.Dp(70),
 		Height:          unit.Dp(20),
+		Padding:         unit.Dp(padding),
 		shaper:          *th.Shaper,
 	}
 }
```

Comment:
I think I do not specifically use the NumericUpDownPadded anymore (see above, I changed my mind), question is whether I should remove the change

Reply:
Padding is still used, but I removed the wrapper calls to the constructor